app/assets/stylesheets/application.css

```diff
@@ -1,3 +1,292 @@
 /*
  *= require font-awesome
  */
+
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;800&family=Poppins:wght@300;400&display=swap');
+
+:root {
+  --primary: #3778c2;
+  --secondary: #5fb523;
+  --text: #434b54;
+}
+
+* {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement previous reviewer's suggestion because you need to work on it, I compare your and [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) designs pictures below🙏🏼


| Your Design  |
| ------------- | 

#
![image](https://user-images.githubusercontent.com/63915024/221375323-ee56640c-af50-497f-98ac-af751797c39e.png)
#

| [Behance Design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)  |
| ------------- |  

#
![image](https://user-images.githubusercontent.com/63915024/221375332-3abf345e-1421-48e1-8ecd-6363942fe047.png)
#